Task 2/Task 2.1/Task 2.1.2/Figures/RoundShapes/Round.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Task_2._1._2.Figures.RoundShapes
+{
+    public class Round : Circle
+    {
+        public Round(Point p, int r) : base(p, r) => Name = "Круг";
+        public override double Area() => Math.PI * Radius * Radius; 
+    }
```

Comment:
Still don't understand why implement Circle **and** Round, when they could be one class. That would be less confusing ("now, which one was it that actually calculates area?..")